homeassistant/components/life360/config_flow.py

```diff
@@ -69,10 +67,16 @@ def async_get_options_flow(config_entry: ConfigEntry) -> Life360OptionsFlow:
 
     async def _async_verify(self, step_id: str) -> FlowResult:
         """Attempt to authorize the provided credentials."""
+        if not self._api:
+            self._api = Life360(
+                session=async_create_clientsession(
```

Comment:
Shouldn't `async_get_clientsession` be used instead?

Reply:
Ah, yes, I think you're right. I used `async_create_clientsession` in coordinator.py because a `ConfigEntry` is associated with it, but here in config_flow.py that's not the case. I guess I'll need to rev the `life360` package and make it specify the timeout in the request as opposed to having it set as a default in the session.

Reply:
Actually, I probably should change the call in coordinator.py, too. It doesn't require a unique session (e.g., it doesn't need its own separate set of cookies), so `async_get_clientsession` should work just fine there as well. I'll add that before pushing a new commit.